app/views/works/show.html.erb

```diff
@@ -5,7 +5,7 @@
 </ul>
 <% if !@work.posted? %>
   <p class="notice">
-    <%= ts("This work is a draft and has not been posted. The draft will be") %> <strong><%= ts("automatically deleted") %></strong> <%= ts("on") %> <%= time_in_zone(@work.created_at + 1.month) %>
+    <%= t(".unposted_deletion_notice", deletion_date: date_in_zone(@work.created_at + 29.days)).html_safe %>
```

Comment:
To fix the Brakeman warning, you can [rename the i18n key](https://guides.rubyonrails.org/i18n.html#using-safe-html-translations) so that it ends in `_html` or `.html`, and then you should be able to delete the call to `html_safe`.

(It might be nice to do that for the message in `works/work_module`, too.)

Reply:
Thanks! I applied your suggestion.